File: server/server/Dto/OrderDto.cs
namespace server.Dto
{
    public class OrderDto
    {
        public long Id { get; set; }
        public DateTime DeliveryTime { get; set; }
        public DateTime OrderTime { get; set; }
        public string DeliveryAddress { get; set; }
        public string DeliveryDescription { get; set; }
        public BuyerDto BuyerDto { get; set; }
        public List<OrderProductAmountDto> OrderProductAmountsDto { get; set; }
    }
}

File: client/src/dto/OrderDto.js
export default class OrderDto{
    constructor(id = null, deliveryTime = null, orderTime = null, deliveryAddress = null, deliveryDescription = null, buyerDto = null, orderProductAmountsDto = null)
    {
        this.Id = id;
        this.DeliveryTime = deliveryTime;
        this.OrderTime = orderTime;
        this.DeliveryAddress = deliveryAddress;
        this.DeliveryDescription = deliveryDescription;
        this.BuyerDto = buyerDto;
        this.OrderProductAmountsDto = orderProductAmountsDto;
    }

    AddProduct(orderProductAmountDto) {
        if (!this.OrderProductAmountDto) {
            this.OrderProductAmountDto = [];
        }
        
        this.OrderProductAmountDto.push(orderProductAmountDto);
    }
}